lib/src/main/scala/spinal/lib/bus/localbus/MemBus.scala

```diff
@@ -0,0 +1,39 @@
+package spinal.lib.bus.localbus
+
+import spinal.core._
+import spinal.lib._
+
+case class MemBusConfig(aw: Int, dw: Int)
+
+case class MemBus(c: MemBusConfig) extends Interface with IMasterSlave {
```

Comment:
MemBus is a near duplicate of the BRAM bus (slight difference in WE, which would actually be a reasonable future feature request to support) and the AsyncMemoryBus (which can stall) - I think we should extend one of those with a parameter instead of adding yet another new Bus that is not reasonably different.